lib/newgistics/default_logger.rb

```diff
@@ -3,6 +3,11 @@
 module Newgistics
   class DefaultLogger
     def self.build
+      Object.const_defined?("Rails") ? Rails.logger : build_logger
```

Comment:
Automatically default to Rails logger if detected, because that's cool in the Rails ecosystem.